docs(uipath-maestro-flow): add shared connection-binding workflow doc (#801)
UiPath/skills: skills/uipath-maestro-flow/references/shared/connection-binding.md

## Standard list + ping

```bash
# 1. List connections for the connector
uip is connections list "<connector-key>" --output json

# 2. Pick the default enabled connection (IsDefault: Yes, State: Enabled)

# 3. Verify health
uip is connections ping "<connection-id>" --output json
```

Add `--folder-key "<folder-key>"` to `connections list` when authoring a connector activity that must bind in a specific folder.

## Empty-list recovery

If `connections list` returns an empty array, retry once with `--refresh` to bypass the CLI cache:

```bash
uip is connections list "<connector-key>" --refresh --output json
```

## No-healthy-connection STOP

If `--refresh` still returns nothing, **STOP**. The node cannot be configured without a real connection ID. Present the path forward via `AskUserQuestion` (see the dropdown rule in [SKILL.md](../../SKILL.md)):

- **Create a new connection** — `uip is connections create "<connector-key>"` starts the OAuth flow; user completes browser auth, then re-run `connections list` to pick up the new connection.

> - **Switch this node to manual mode** — applies to HTTP nodes only.

Reviewer: I don't love this instruction, not clear enough to disambiguate.

- **Skip this node** — leave it in the flow with empty `inputs`; document it as an open question for the user.
- **Something else** — free-form input.

> Do not fall back silently, invent a placeholder ID, or skip the node without explicit user selection.

Reviewer: ```suggestion
**NEVER** fall back silently, invent a placeholder ID, or skip the node without explicit user selection.
```
